### PR TITLE
feat(cbr): cbr_vault support update enterprise_project_id

### PR DESCRIPTION
**Pull request by wangyuancheng01**
**What this PR does / why we need it**: cbr_vault support update enterprise_project_id

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cbr" TESTARGS="-run TestAccVault_withEpsId"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cbr -v -run TestAccVault_backupServer -timeout 360m -parallel 4
=== RUN   TestAccVault_backupServer
=== PAUSE TestAccVault_backupServer
=== CONT  TestAccVault_backupServer
--- PASS: TestAccVault_backupServer (479.44s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cbr       479.495s
```

### CODE REVIEW

File: docs/resources/cbr_vault.md [hunk 1]
```diff
@@ -220,8 +220,8 @@ The following arguments are supported:
 
 * `bind_rules` - (Optional, Map) Specifies the tags to filter resources for automatic association with **auto_bind**.
 
-* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the ID of the enterprise project to which the vault
-  belongs. Changing this will create a new vault.
+* `enterprise_project_id` - (Optional, String) Specifies the ID of the enterprise project to which the vault
+  belongs.
```
Comment (Lance52259) [inline docs/resources/cbr_vault.md]:
```suggestion
* `enterprise_project_id` - (Optional, String) Specifies the ID of the enterprise project to which the vault belongs.
```
ensure that the text is neat during modification.

File: huaweicloud/services/acceptance/cbr/resource_huaweicloud_cbr_vault_test.go [hunk 2]
```diff
@@ -172,7 +174,7 @@ resource "huaweicloud_cbr_vault" "test" {
 `, basicConfig, name)
 }
 
-func testAccVault_backupServer_step2(basicConfig, name string) string {
+func testAccVault_backupServer_step2(basicConfig, name, epsId string) string {
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/cbr/resource_huaweicloud_cbr_vault_test.go]:
```suggestion
func testAccVault_backupServer_step2(basicConfig, name string) string {
```
there is unnecessary to append a new argument, and we can use the env in the script as follows:
```
enterprise_project_id = %[3]s
...

`, basicConfig, name, acceptance.HW_ENTERPRISE_PROJECT_ID_TEST)
```

Comment (wangyuancheng01) [reply huaweicloud/services/acceptance/cbr/resource_huaweicloud_cbr_vault_test.go]: good idea, this suggestion is very nice

File: docs/resources/cbr_vault.md (hunk 1 above)
Comment (wangyuancheng01) [reply docs/resources/cbr_vault.md]: ok, I will pay attention to it next.

**Pull request merged**